lingetic-spring-backend/docker-compose.yml

```
@@ -10,5 +10,17 @@ services:
+      - RABBITMQ_DEFAULT_PASS=${RABBITMQ_PASSWORD:?RABBITMQ_PASSWORD environment variable is not set}
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    volumes:
+      - rabbitmq_data:/var/lib/rabbitmq
+
 volumes:
-  pgdata:
\ No newline at end of file
+  pgdata:
+  rabbitmq_data:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)